src/main/java/no/unit/nva/database/DatabaseServiceImpl.java

```diff
@@ -128,6 +142,15 @@ public RoleDto getRole(RoleDto queryObject) throws NotFoundException, InvalidEnt
         return convertQueryResultToOptionalUser(userSearchResult, queryObject);
     }
 
+    private DynamoDBQueryExpression<UserDb> createListUsersQuery(String institution)
+        throws InvalidEntryInternalException {
+        UserDb queryObject = UserDb.newBuilder().withUsername(NOT_USED).withInstitution(institution).build();
```

Comment:
Do we need to call withUsername() at all?

Reply:
Yes, because if we do not use a username the object cannot be build. It fails.

Reply:
Burde ikke det vært håndtert i Builder og ikke her?

Reply:
It is the builder that throws the exception, so that we cannot create  a user without a username